docs/fields.asciidoc

```diff
@@ -1512,6 +1512,18 @@ type: long
 
 --
 
+
+
+*`span.servicemap.hash.xxhash`*::
```

Comment:
Do consumers care about the hash details? What was the reasoning behind changing from `span.servicemap.fingerprint`? IMO we don't need to explain the implementation details here - it's just an opaque hash.

Reply:
I generally agree, but tried to be ECS compliant with the naming https://github.com/elastic/ecs/blob/master/docs/field-details.asciidoc#hash-fields, which lead to me this name, now that we are using a hash algo. 
The point where I was uncertain is whether or not `64-bit` information should be contained, but I decided against it as this could be seen the default.

Reply:
Hmm. In the ECS cases they're not opaque hashes, though - they're being used for integrity checks. Also, there's `event.hash` which doesn't specify the type of hash.

I suppose it doesn't matter _too_ much as Kibana and APM Server release together, but I still think it'd be better to not identify the implementation details. That would allow us to change the algorithm used later, without affecting the UI.

Reply:
+1 on opaque approach

Reply:
Concur with @axw. The other hashes in ECS are either for integrity or unique identification of an object (e.g. a file). The `event.hash` is a hash of the entirety of the original event. This fingerprint approach for hashing a couple of fields together is common in the infosec community too for easier pivoting in Kibana. For this usecase, it seems `fingerprint` might be more appropriate.

Reply:
Thanks for all the input, I'll change accordingly.